src/Model/GamePackage/Game.java

```diff
@@ -1,4 +1,557 @@
 package Model.GamePackage;
 
+import Model.BoardPackage.BoardSquare;
+import Model.BoardPackage.OwnableSquare;
+import Model.CardPackage.Card;
+import Model.CardPackage.Deck;
+import Model.BoardPackage.PropertySquare;
+import Model.BoardPackage.Board;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**********************************************************************
+ * The Game class is responsible for almost all of the game logic.
+ * Any action that goes on in the game is handled here.
+ *
+ * @author Dylan Kernohan
+ * @version 2/19/2018
+ *********************************************************************/
 public class Game {
```

Comment:
Is there going to be a separate pull request for the game class specifically or should I review it here in this pull request?

Reply:
Crap. Yeah that was not supposed to be in this. I will try and remove it